src/components/Skeleton/view/Skeleton.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+import { StyledSkeleton } from './style';
+import { SkeletonProps } from '../types';
+
+export const Skeleton: FC<SkeletonProps> = ({ ...props }) => {
```

Comment:
- `Skeleton = StyledSkeleton` v tomto případě udělá to samé
- Pokud propy nerozbaluju, můžu nechat jenom jako `(props) =>`
- Defaultní propy definovat v parametru `({ shape = "rectangle", ...props }) =>`

Ani jedna věc není nijak zásadní problém, mergnu to takto prozatím